build/media-suite_compile.sh

```diff
@@ -1182,6 +1182,27 @@ if [[ $ffmpeg != no ]] && enabled liblc3 &&
     do_checkIfExist
 fi
 
+_check=(bin/atw_ldwrapper libAudioToolboxWrapper.a
+    bin-video/{ASL,CoreAudioToolbox,CoreFoundation,icudt62,libdispatch,libicuin,libicuuc,objc}.dll)
+if [[ $ffmpeg != no ]] && enabled audiotoolbox &&
+    do_vcs "$SOURCE_REPO_AUDIOTOOLBOX"; then
+    _qtfiles_url="https://github.com/AnimMouse/QTFiles/releases/download/v12.10.11"
+    do_uninstall "${_check[@]}"
+    if [[ $build64 = yes ]]; then
+        do_wget -c -r -q "${_qtfiles_url}/QTfiles64.7z" QTfiles64.7z
+        do_install QTfiles64/*.dll bin-video
+        rm -rf QTfiles64/ QTfiles64.7z
+    fi
+    if [[ $build32 = yes ]]; then
+        do_wget -c -r -q "${_qtfiles_url}/QTfiles.7z" QTfiles.7z
```

Comment:
Please change the do_wget lines to remove the `-c` and add a hash to check against with the `-h` argument, preferable a sha256 one. This can help weed out issues regarding internet connection and avoid redownloading if the file is already present. This can help in cases where a user nukes their local64/local32 folder but kept the build folder intact.